app/api/routers/attributes.py

```diff
@@ -1,12 +1,27 @@
-from fastapi import APIRouter
+from fastapi import APIRouter, Request
 from pydantic import constr
 
 from .. import crud
-from ..models import CONTROLLED_TERM_REGEX
+from .. import utility as util
+from ..models import CONTROLLED_TERM_REGEX, DataElementURI, VocabLabelsResponse
 
 router = APIRouter(prefix="/attributes", tags=["attributes"])
 
 
+@router.get("/{data_element_URI}/vocab", response_model=VocabLabelsResponse)
+async def get_term_labels_for_vocab(
+    data_element_URI: DataElementURI, request: Request
+):
+    """When a GET request is sent, return a dict containing the name, namespace info, and all term ID-label mappings for the vocabulary of the specified variable."""
+    if data_element_URI is DataElementURI.assessment:
+        response = await crud.get_term_labels_for_cogatlas(
+            term_labels_path=request.app.state.vocab_dir_path
+            / f"{util.VOCAB_FILE_PREFIX['cogatlas']}_term_labels.json"
```

Comment:
I think the readability of this line suffers more than necessary from this future-proofing/configurability. My suggestion would be either

- store the actual filepath on the state object (see main.py comment) OR
- hardcode the path here since this section is only concerned with cogatlas